lib/landrush/action/common.rb

```diff
@@ -71,7 +71,7 @@ def read_machine_hostname
           return machine.config.vm.hostname
         end
 
-        "#{Pathname.pwd.basename}.#{config.tld}"
+        "#{Pathname.pwd.basename}.#{config.tld[0]}"
```

Comment:
>     config.landrush.host 'two.org' true

So I would _still_ need 'config.landrush.host'? But now with a boolean that is less self-documenting.

This is more clear to me (and perhaps only me :-) ) because simply builds on existing syntax:

```
config.landrush.tld = [ 'one.org', 'two.org', ... ]
config.landrush.tld.each do |tld|
  config.landrush.host tld
end
```

> sudo cp -r /etc/resolver/one.org /etc/resolver/two.org

Sure, there are many ways to achieve the end goal and going outside landrush to manage extra TLDs is one of them, but that seems less clean to me.